test/conformance/testing/include/uur/fixtures.h

```diff
@@ -435,17 +435,23 @@ std::string deviceTestWithParamPrinter(
     return uur::GetPlatformAndDeviceName(device) + "__" + ss.str();
 }
 
-struct urProgramILBinaryTest : urContextTest {
+struct urProgramTest : urContextTest {
     void SetUp() override {
         UUR_RETURN_ON_FATAL_FAILURE(urContextTest::SetUp());
-        uur::KernelsEnvironment::instance->LoadSource("nop", 0, il_binary);
+        uur::KernelsEnvironment::instance->LoadSource("foo", 0, il_binary);
```

Comment:
It might be nice if we could parametrize this for all the sycl programs you have - not sure how you would be able to do this though.
Maybe we can do something with `add_device_binary()`, but not 100% sure.

Anyway - it's not required for this PR so just ignore my rambling :)

Reply:
it does seem like a prime candidate for parameterization